developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -80,26 +102,168 @@ You can request additional properties by sending a request body with the :code:`
+- Property: ``<nc:is-encrypted />``
+- Description: Whether the file is end-to-end encrypted
+
+favorite
+^^^^^^^^
+- Id: ``{http://owncloud.org/ns}favorite``
+- Property: ``<oc:favorite />``
+- Description: The favorite state.
+- Example: ``0`` for not favourited, ``1`` for favourited
+
+comments-href
+^^^^^^^^^^^^^
+- Id: ``{http://owncloud.org/ns}comments-href``
+- Property: ``<oc:comments-href />``
+- Description: The DAV endpoint to fetch teh comments
+
+comments-count
+^^^^^^^^^^^^^^
+- Id: ``{http://owncloud.org/ns}comments-count``
+- Property: ``<oc:comments-count />``
+- Description: The number of comments
+
+comments-unread
+^^^^^^^^^^^^^^^
+- Id: ``{http://owncloud.org/ns}comments-unread``
+- Property: ``<oc:comments-unread />``
+- Description: The number of unread comments
+
+owner-id
+^^^^^^^^
+- Id: ``{http://owncloud.org/ns}owner-id``
+- Property: ``<oc:owner-id />``
+- Description: The user id of the owner of a shared file
+
+owner-display-name
+^^^^^^^^^^^^^^^^^^
+- Id: ``{http://owncloud.org/ns}owner-display-name``
+- Property: ``<oc:owner-display-name />``
+- Description: The display name of the owner of a shared file
+
+share-types
+^^^^^^^^^^^
+- Id: ``{http://owncloud.org/ns}share-types``
+- Property: ``<oc:share-types />``
+- Description: The list of type share.
+- Example: ``0`` = user; ``1`` = group; ``3`` = public link; ``4`` = email; ``6`` = federated cloud share; ``7`` = circle; ``10`` = Talk conversation
+
+share-permissions
+^^^^^^^^^^^^^^^^^
+- Id: ``{http://open-collaboration-services.org/ns}share-permissions``
+- Property: ``{http://open-collaboration-services.org/ns}share-permissions``
+- Description: The permissions that the user has over the share.
+- Example: Create = ``4``; Read = ``1``; Update = ``2``; Delete = ``8``; Share = ``16``; All = ``31``;
+
+checksums
+^^^^^^^^^
+- Id: ``{http://owncloud.org/ns}checksums``
+- Property: ``<oc:checksums />``
+- Description:
+has-preview
+^^^^^^^^^^^
+- Id: ``{http://nextcloud.org/ns}has-preview``
+- Property: ``<nc:has-preview />``
+- Description: Whether a preview of the file is available
+
+size
+^^^^
+- Id: ``{http://owncloud.org/ns}size``
+- Property: ``<oc:size />``
+- Description: Unlike :code:`getcontentlength`, this property also works for folders reporting the size of everything in the folder
+
+rich-workspace
+^^^^^^^^^^^^^^
+- Id: ``{http://nextcloud.org/ns}rich-workspace``
+- Property: ``<nc:rich-workspace />``
+- Description: This property is provided by the text app
+
+contained-folder-count
+^^^^^^^^^^^^^^^^^^^^^^
+- Id: ``{http://nextcloud.org/ns}contained-folder-count``
+- Property: ``<nc:contained-folder-count />``
+- Description: The number of folders directly contained in the folder (not recursively)
+
+contained-file-count
+^^^^^^^^^^^^^^^^^^^^
+- Id: ``{http://nextcloud.org/ns}contained-file-count``
+- Property: ``<nc:contained-file-count />``
+- Description: The number of files directly contained in the folder (not recursively)
```

Comment:
I'd prefer a markdown table because the information will look a lot more structured.

Reply:
I tried a table, but it is unreadable as the container's width is too small. Meaning, we only see the id, and you have to scroll right to read the remaining text.

Reply:
Ah well :/

Reply:
Found a solution to split long text in multiple lines. So this is a table now :)